src/test/kotlin/org/rust/ide/intentions/UnwrapLambdaExprIntentionTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.rust.ide.intentions
+
+import org.rust.lang.RustTestCaseBase
+
+class UnwrapLambdaExprIntentionTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/ide/intentions/fixtures/unwrap_lambda_expr/"
+
+    fun testEnableUnwrapBraces() = checkByFile {
+        openFileInEditor("enable_unwrap_braces.rs")
+        myFixture.launchAction(UnwrapLambdaExprIntention())
+    }
+
+    fun testUnableUnwrapBraces() = checkByFile {
+        openFileInEditor("unable_unwrap_braces.rs")
+        myFixture.launchAction(UnwrapLambdaExprIntention())
+    }
```

Comment:
i prefer to use unable, it means we can not trigger this intention. anyway, enabled/disabled also make sense.